src/compilation/compiler.rs

```diff
@@ -121,10 +143,16 @@ impl Compiler {
         }
     }
 
+    /// Assuming [compile()][a] was called, this function prints the generated LLVM.
```

Comment:
This can theoretically be called without calling compile first I think, but I'm pretty sure it would just be an empty file then. Do we want to remove the `Assuming compile() was called` part?

Reply:
Agreed. I flipped it to say

```suggestion
    /// This function prints the LLVM IR generated so far (via methods like [compile][a]).
```